src/rest/SearchEndpoint.ts

```diff
@@ -340,7 +340,8 @@ export class SearchEndpoint implements ISearchEndpoint {
     callOptions?: IEndpointCallOptions,
     callParams?: IEndpointCallParameters
   ) {
-    const call = this.buildCompleteCall(options, callOptions, callParams);
+    const token = options.handshakeToken;
+    const call = this.buildCompleteCall({ token }, callOptions, callParams);
```

Comment:
@acote-coveo I'm reverting the param name to `token` for now to match the search-api endpoint in production.